lib/vttcue.js

```diff
@@ -68,27 +99,28 @@
     var _size = 50;
     var _align = "middle";
 
-    Object.defineProperties(this, {
-      "id": {
-        enumerable: true,
+    Object.defineProperty(cue,
```

Comment:
Is this solving the enumerable issue that I described earlier for `toJSON` to work during the tests?